addons/web/static/src/legacy/scss/ui.scss

```diff
@@ -95,6 +95,26 @@ span.o_force_ltr {
     direction: ltr;
 }
 
+// == Inline avatar image
+//------------------------------------------------------------------------------
+// Commonly used for avatar images rendered within text.
+// Its size is relative to the current font-size.
+
+.o_avatar_inline {
+    width: $o-avatar-inline-size;
+    height: $o-avatar-inline-size;
+    object-fit: cover;
+    vertical-align: text-bottom;
+}
+
+// == Thread avatar image
+//------------------------------------------------------------------------------
+.o_avatar_thread {
+    width: $o-avatar-thread-size;
+    height: $o-avatar-thread-size;
+    object-fit: cover;
+}
```

Comment:
The concept of "thread" does not exist in the `web` module. This should either be defined in `mail`, or find another name.

Reply:
The `thread` name and  the `web` locations seemed ok to me  because:
- the class is used by more than one module (`mail`and `im_livechat`)
- both occurrences are in "threads" content

Any advice?

Reply:
There is a dependency on `mail` for all modules that use "thread, including `im_livechat`, so the move to `mail` would be ok in that regard.

Reply:
Ok, I'll move it to `mail` and I'll use the  `o_mail_ thread_avatar ` 👍